Source/OxyPlot/Series/HeatMapSeries.cs

```diff
@@ -122,6 +122,21 @@ public HeatMapSeries()
         public bool Interpolate { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to use discrete rectangles when rendering (ignored if Interpolate is set to true). The default value is <c>false</c>.
+        /// </summary>
+        public bool RenderDiscreteRectangles { get; set; }
```

Comment:
Ah, yes, I suppose if we may have more than two rendering modes, that would make sense. I would be in favour of creating an enum, given just how much code is shared that has nothing to do with rendering.

Image rending does (by my reckoning) work with the log axes, I attach an interpolated version of the logarithmic dataset PDF supplied below.

[interpolatedlogxyheatmap.pdf](https://github.com/oxyplot/oxyplot/files/150190/acctest_2img.pdf)